src/diffusers/pipelines/stable_diffusion/pipeline_onnx_stable_diffusion.py

```diff
@@ -213,8 +192,8 @@ def _encode_prompt(self, prompt, num_images_per_prompt, do_classifier_free_guida
     def __call__(
         self,
         prompt: Union[str, List[str]],
-        height: Optional[int] = None,
-        width: Optional[int] = None,
+        height: Optional[int] = 512,
+        width: Optional[int] = 512,
```

Comment:
No model config to infer the sizes from -> returning the defaults for now

Reply:
The conversion script fetches the image shape from `pipeline.vae.config.sample_size` for tracing, so that should help for now